tools/rollup-es3/src/es3/Es3Tokens.ts

```diff
@@ -198,6 +198,14 @@ export const defaultEs3CheckTokens:IEs3CheckKeyword[] = [
             "Util.trim",                            // Make sure this isn't a reference to Util.trim()
             "DataSanitizer.trim"                    // Make sure this isn't a reference to Util.trim()
         ]
+    },
+    {
+        funcNames: [ /([\w0-9\$]*)(\(\))?\.now[\s]*\(\)/g ],
```

Comment:
What is this rollup-es3 tool doing?, is automatically executed?

Reply:
The rollup-es3 is a packaging check that does
- Checks that non supported es3 functions are not used (and blocks the packaging step if any are detected) -- this is how I found all of the Date.now() usages
- For a small subset of functions (mostly created by rollup itself) it replaces non-es3 functions with inline polyfill versions.